pkg/manager/index/service/option.go

```diff
@@ -34,6 +34,8 @@ var (
 		WithIndexingConcurrency(1),
 		WithIndexingDuration("1m"),
 		WithIndexingDurationLimit("30m"),
+		WithMinUncommitted(100),
+		WithAgentNamespace("default"),
```

Comment:
Also I forgot implementation about node selecting, could you add it? same as namespace